pybossa/view/projects.py

```diff
@@ -813,6 +813,156 @@ def details(short_name):
     return handle_content_type(response)
 
 
+def update_data(body, project):
+    if "project" in body:
+        key = "project"
+        data = body.get(key) or {}
+        external_config = project.info.get('ext_config') or {}
+        target_bucket = data.get('target_bucket')
+        if not external_config:
+            gigwork_poller = dict(target_bucket=target_bucket)
```

Comment:
the contents of `external_config` are configuration driven, `target_bucket` and `gigwork_poller` are never hard-coded.

Reply:
agree, will push another commit to fix.